src/anomalib/models/cfa/config.yaml

```diff
@@ -62,7 +62,7 @@ logging:
   log_graph: false # Logs the model graph to respective logger.
 
 optimization:
-  export_mode: null #options: onnx, openvino
+  export_mode: null # options: torch, onnx, openvino
```

Comment:
What if a user would like to export to both torch and openvino/onnx format?

Reply:
This will change with the new CLI. In this case, we would only add a temporary change to this.

Reply:
Alright, let's leave this for later then